recipes/hall-lab-svtools/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: hall-lab-svtools
+  version: "0.1.1"
+
+source:
+  fn: svtools-3731a7c6e19.tar.gz
+  url: https://github.com/hall-lab/svtools/archive/3731a7c6e19.tar.gz
+  md5: c6ba71cd3a7f02b0c58d1d13a48c02db
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
```

Comment:
I think setuptools should be only a build requirement. Or do they use the setuptools API for something at runtime?